docs/content/manual/release-specific/v1.2.0/test-backing-image-upload.md

```diff
@@ -14,6 +14,25 @@ title: Test backing image
 9. Create and attach a volume with the backing image. Verify the data content is correct.
 10. Do cleanup.
 
+### Upload via Rancher URI
+<sup>Related issue: [3129](https://github.com/longhorn/longhorn/issues/3129) with fix in Longhorn v1.3.0+</sup>
+1. With Rancher v2.6.x create a cluster and install Longhorn
+   - or import an existing cluster with Longhorn installed
+2. Go into the cluster
+3. Click Longhorn from the Rancher menu, the sidebar on the left
+4. Click on the Longhorn app to open the UI
+   - An example URL: <small>`https://rancer.server.domain/k8s/clusters/CLUSTER-ID/api/v1/namespaces/longhorn-system/services/http:longhorn-frontend:80/proxy/#/backingImage`</small>
+5. Create a new backing image with **Upload From Local**
```

Comment:
I would recommend testing with a large backing image (> 1Gi).

Reply:
These two cases were extended from the main test, already mentioned using a large image file. I think it will be okay to keep these two sub-cases minimal.